ROADMAP.md

```diff
@@ -0,0 +1,359 @@
+- Effort: approx. 6 months
+
+**Why:** Testground provides an excellent foundation for distributed testing. Supporting the libp2p team with interoperability tests generates long-term value outside of Testground. We can move faster, generate interest, and create measurable improvements by focusing on a use-case.
+
+**Deliverables:**
+
+- Tooling to support interoperability testing (mixed-builders, composition templating),
+- Stability measurements & fixes to reach the libp2p team's expectations,
+- A fully working example (ping test) used in go-libp2p and rust-libp2p CIs,
+- An interoperability Dashboard that shows how implementations and versions are tested.
+
+### 2. Refresh Testground's EKS support
+
+- Delivery: Q4 2022
+- Theme: usefulness
+- Effort: approx. 6 months
+
+**Why:** Testground can simulate small networks in CI, but it covers more use cases when it lives in a larger cluster. When we run Testgroun in Kubernetes, we can support whole organizations through the Testground As A Service product.
+
+Using a managed service (Amazon's Elastic Kubernetes Service) means our maintenance costs are lower, and the team can focus on improvements.
+
+**Deliverables:**
+
+- An EKS installation script,
+  - Extra care is taken on Network infrastructure (CNIs).
+- A (fixed) Kubernetes runner that runs on Amazon's EKS,
+- The team can use the latest Testground version, create new releases, and upgrade the cluster.
+
+### 3. Improve our Development & Testing infrastructure to meet engineering standards
+
+- Delivery: Q1 2023
+- Theme: reliability & sustainability
+- Effort: approx. 4 months
+
+**Why:** Testground proved itself valuable multiple times over the years. However, now we need bulletproof development processes to make the project sustainable and facilitate external contributions.
+
+Extra care is taken on Testing and Stability: we are building a testing platform, and Testground's testing must be impeccable.
+
+**Deliverables:**
+
+- Automated release process with official releases, binary distribution, and Changelogs - [EPIC 1430](https://github.com/testground/testground/issues/1430)
+- Well-defined Project management processes like triaging, contribution & reviewing processes, etc.
+- Documentations for maintainers,
+- Maintainable integration testing tooling (no more shell scripts or flakiness),
+- A Stability Dashboard used to identify regression & discuss improvement with Maintainers and Users,
+- Tooling for EKS testing.
+
+### 4. Provide a Testground As A Service Cluster used by libp2p & ipfs teams
+
+- Delivery: Q3 2023
+- Theme: usefulness
+- Effort: approx. 4 months
+
+**Why:** Improving experience with faster testing in user's CI. Cover more use cases with large-scale networks.
+
+**Deliverables:**
+
+- A _stable_ cluster,
+- Authentication,
+- Tooling for users to use EKS cluster in their testing,
+- Integration of the EKS feature in our testing infrastructure
+  - Short-lived clusters used during integration testing
```

Comment:
With ~30 minute setup and no support for multiple clusters in the same region, I think we're a long way from that.

Reply:
(resolved)